tests/chainer_tests/test_function_node.py

```diff
@@ -920,6 +918,11 @@ def use_gpu(self):
             if isinstance(value, chainer.Variable):
                 value.to_gpu()
 
+    def use_chainerx(self, device='native:1'):
```

Comment:
Why `native:1` rather than `native:0`? 
How about refactoring the test to use `inject_backend_tests`?